Packs/ApiModules/Scripts/TAXII2ApiModule/TAXII2ApiModule_test.py

```diff
@@ -2287,3 +2273,23 @@ def test_create_x509_certificate_object():
         "issuer": "C=ZA, ST=Western Cape, L=Cape Town, O=Thawte Consulting cc, OU=Certification Services Division,"
         " CN=Thawte Server CA/emailAddress=server-certs@thawte.com",
     }
+
+
+@pytest.mark.parametrize(
+    "pattern, value",
+    [
+        ("[domain-name:value = 'www.example.com']", 'www.example.com'),
+        ("[file:hashes.'SHA-256' = 'xxxxxxxxxxxx']", 'xxxxxxxxxxxx'),
```

Comment:
Please use either a random hash or just zeros in the correct length, same for the MD5.

Reply:
Done